utils/DeployContract.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
+// TODO Gulshan: Documentation
+// TODO Gulshan: class format
```

Comment:
Ticket create to address it :-
https://github.com/OpenSTFoundation/openst.js/issues/54